README.md

```diff
@@ -1,5 +1,26 @@
 # mlh-final-project
 
+## Installation
+Make sure python3 and pip are installed. 
+
+Create a virtual environment
+
+``$ python3 -m venv virtualenv``
+
+``$ source virtualenv/bin/activate``
+
+Switch to flask directory and install requirements
+
+``$ cd flask``
+
```

Comment:
I recommend having `pip install -r requirements.txt` instead of installing packages separately